### PR TITLE
Add iOS support to ani-cli

### PR DESCRIPTION
**Pull request by Lockl00p**
Pull Request Template
Type of change
[ ] Bug fix
[X] Feature
[ ] Documentation update

Description
added iSH detection, and used x-callback (Thanks [RubenPonce](https://github.com/RubenPonce) for telling me about that!) when iSH is detected. Oh, also adjusted readme to include iOS installation instructions.

Checklist
[X] any anime playing
[X] bumped version
---
[X] next, prev and replay work
[X] -c history and continue work
[X] -d downloads work
[X] -s syncplay works (Not on iOS)
[X] -q quality works
[X] -v vlc works
[X] -e select episode works
[X] -S select index works
[X] -r range selection works
[X] --dub both work
[X] all providers return links (not necessarily on a single anime, use debug mode to confirm)
---
[X] -h help info is up to date
[X] Readme is up to date
[X] Man page is up to date

Additional Testcases
The safe bet: One Piece
Episode 0: Saenai Heroine no Sodatekata ♭
Unicode: Saenai Heroine no Sodatekata ♭
Non-whole episodes: Tensei shitara slime datta ken (ep. 24.5, ep. 24.9)

### CODE REVIEW

**Comment by port19x** [comment]
This is already looking a lot better!
I'll do a proper review sometime tomorrow

**Review by Lockl00p — COMMENTED** [review]
It worked!

File: README.md [hunk 1]
```diff
@@ -212,6 +213,18 @@ rm -rf ani-cli
 
 For players you can use the apk (playstore/fdroid) versions of mpv and vlc. Note that these cannot be checked from termux so a warning is generated when checking dependencies.
 
+### iOS
+Install VLC from app store
```
Comment (port19x) [inline README.md:217]: What's the reason for vlc by default again?

File: ani-cli [hunk 2]
```diff
@@ -231,6 +231,7 @@ play_episode() {
         *yncpla*) nohup "$player_function" "$episode" -- --force-media-title="${allanime_title}episode-${ep_no}-${mode}" >/dev/null 2>&1 & ;;
         download) "$player_function" "$episode" "${allanime_title}episode-${ep_no}-${mode}" ;;
         catt) nohup catt cast "$episode" >/dev/null 2>&1 & ;;
+	    iSH) printf "\e]8;;vlc-x-callback://x-callback-url/stream?url=%s&filename=%sepisode-%s-%s\a~~~~~~~~~~~~~~~~~~~~\n~ Tap to open VLC ~\n~~~~~~~~~~~~~~~~~~~~\e]8;;\a\n" "$episode" "$allanime_title" "$ep_no" "$mode";sleep 5 ;;
```
Comment (port19x) [inline ani-cli:234]: A bit hacky, but I trust you that vlc can't be opened non-interactively from iSH

File: README.md (hunk 1 above)
Comment (71zenith) [reply README.md:217]: > What's the reason for vlc by default again?

Mpv isnt available on iOS

**Review by port19x — APPROVED** [review]
Good code :+1:

**Pull request merged**